packages/entity-server/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "allowUnreachableCode": false,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "CommonJS",
+    "noEmit": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "target": "ES2019",
+    "typeRoots": ["src/@types", "node_modules/@types"]
+  },
+  "exclude": ["node_modules", "src/@types"]
+}
```

Comment:
Will migrate this into common `tsconfig-ts.dependency.json` as part of https://github.com/beyondessential/tupaia-backlog/issues/2400

Didn't want to make too many cross package changes in this PR